lib/ocpp/v16/enums.cpp

```diff
@@ -1,253 +1,272 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
-#include <ocpp1_6/enums.hpp>
+// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+#include <ocpp/v16/enums.hpp>
 #include <stdexcept>
 #include <string>
 
-namespace ocpp1_6 {
+namespace ocpp {
+namespace v16 {
 
-// from: AuthorizeResponse
+// from: TriggerMessageRequest
```

Comment:
There are a lot of changes here that probably come from the changes sorting in the code generator. Is this sorting better than the old version? If not I would suggest we have another look